include/lsst/meas/modelfit/CModel.h

```diff
@@ -442,6 +449,8 @@ struct CModelResult {
     PTR(afw::detection::Footprint) initialFitRegion;  ///< Pixels used in the initial fit.
     PTR(afw::detection::Footprint) finalFitRegion;    ///< Pixels used in the exp, dev, and linear fits.
 
+    LocalUnitTransform fitSysToMeasSys; ///< Transforms to the coordinate system where parameters are defined
```

Comment:
The confusion here is due to our naming conventions, which require lower-case letters for the start of variables: "FitSys" is the name of one of the coordinate systems, and "MeasSys" is the other, and this is a coordinate transformation object that goes between those two systems.